src/util.rs

```diff
@@ -926,6 +926,37 @@ pub fn daemonize() -> Result<()> {
     Ok(())
 }
 
+#[cfg(any(feature = "dist-server", feature = "dist-client"))]
+mod reqwest_dns_resolver {
```

Comment:
What about moving it to the dedicated `.rs` file?

Reply:
Try [reqwest-hickory-resolver](https://crates.io/crates/reqwest-hickory-resolver) that maintained by me :smile: